static/app/utils/withExperiment.spec.jsx

```diff
@@ -1,80 +1,20 @@
-import {fireEvent, render, screen} from 'sentry-test/reactTestingLibrary';
+import {render, screen} from 'sentry-test/reactTestingLibrary';
 
-import ConfigStore from 'sentry/stores/configStore';
-import {logExperiment} from 'sentry/utils/analytics';
 import withExperiment from 'sentry/utils/withExperiment';
 
-jest.mock('sentry/utils/analytics', () => ({
-  logExperiment: jest.fn(),
-}));
-
-jest.mock('sentry/data/experimentConfig', () => ({
-  experimentConfig: {
-    orgExperiment: {
-      key: 'orgExperiment',
-      type: 'organization',
-      parameter: 'exposed',
-      assignments: [1, 0, -1],
-    },
-    userExperiment: {
-      key: 'userExperiment',
-      type: 'user',
-      parameter: 'exposed',
-      assignments: [1, 0, -1],
-    },
-  },
-}));
-
 describe('withConfig HoC', function () {
   beforeEach(function () {
     jest.clearAllMocks();
   });
 
-  const organization = TestStubs.Organization({
-    id: 1,
-    experiments: {orgExperiment: 1},
-  });
-
   function MyComponent(props) {
     return <span>{props.experimentAssignment}</span>;
   }
-  function TriggerComponent(props) {
-    return <button onClick={props.logExperiment}>click me</button>;
-  }
-
-  it('injects org experiment assignment', function () {
```

Comment:
Removed some of these test cases because it is already tested in `useExperiment.spec` in getsentry, this simply converts it to an HoC